CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
-cmake_minimum_required (VERSION 3.12)
+cmake_minimum_required (VERSION 3.15)
```

Comment:
Maybe you also need to update here https://github.com/Azure/azure-sdk-for-cpp/blob/master/README.md#prerequisites

Reply:
Nice catch, this is changed so I'll resolve this comment.